batchglm/train/numpy/base_glm/estimator.py

```diff
@@ -86,8 +94,95 @@ def train_sequence(self, training_strategy, **kwargs):
                         "overrding %s from training strategy with value %s with new value %s\n"
                         % (x, str(d[x]), str(kwargs[x]))
                     )
+            if "dispersion_smoothing" in d:
+                if d["dispersion_smoothing"] == "sctransform":
+                    if FDataGrid is None or NWSmoother is None:
+                        logger.error("Missing optional dependency scikit-fda.")
             self.train(**d, **kwargs)
             logger.debug("Training sequence #%d complete", idx + 1)
+            if "dispersion_smoothing" in d:
+                if d["dispersion_smoothing"] == "sctransform":
```

Comment:
same for `d["dispersion_smoothing"] == "sctransform"`

Reply:
see above